### ISSUE TITLE
Plugin Fails on Elasticsearch 5.0.0 Alpha 3

### ISSUE THREAD

**Issue by petercort** [s12v/newrelic-elasticsearch#41]
Output from the daemon.log

```
SEVERE: An error has occurred
java.lang.NullPointerException
    at me.snov.newrelic.elasticsearch.reporters.NodesStatsReporter.reportThreadPool(NodesStatsReporter.java:127)
    at me.snov.newrelic.elasticsearch.reporters.NodesStatsReporter.reportNodeStats(NodesStatsReporter.java:87)
    at me.snov.newrelic.elasticsearch.reporters.NodesStatsReporter.reportNodesStats(NodesStatsReporter.java:24)
    at me.snov.newrelic.elasticsearch.ElasticsearchAgent.pollCycle(ElasticsearchAgent.java:44)
    at com.newrelic.metrics.publish.Runner$PollAgentsRunnable.run(Runner.java:209)
    at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
    at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308)
    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180)
    at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294)
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
    at java.lang.Thread.run(Thread.java:745)

```

Config is vanilla, and the newrelic_plugin.log isn't reporting anything. Verified that the api key is correct.

**Comment by s12v** [comment]
yes, it fails on ES 5.x alpha: https://travis-ci.org/s12v/newrelic-elasticsearch/builds/139254603. I will look into it

**Comment by tylerturk** [comment]
The PR that was merged appears to still be failing the node stats parser test. Is there an alpha release of this I could test / install via npi?

**Comment by s12v** [comment]
@tylerturk, I've published 2.3.0

**Comment by tylerturk** [comment]
Thanks @s12v! I have confirmed it works on 5.1.1. 👍
